### ISSUE TITLE
Publication of the flutter package fails

### ISSUE THREAD

**Issue by HofmannZ** [zino-app/graphql-flutter#299]
The publication of the `graphql_flutter` package fails because of `Your pubspec.yaml must not override non-dev dependencies`.

The full output:
```
Missing requirements:
* Your pubspec.yaml must not override non-dev dependencies.
  This ensures you test your package against the same versions of its dependencies
  that users will have when they use it.
Suggestions:
* ./CHANGELOG.md doesn't mention current version (1.1.0).
  Consider updating it with notes on this version prior to publication.
Sorry, your package is missing a requirement and can't be published yet.
For more information, see: https://www.dartlang.org/tools/pub/cmd/pub-lish.
Exited with code 65
```

**Comment by micimize** [comment]
This is because I added 
```yaml
dependency_overrides:
  graphql:
    path: ../graphql
```
Pretty annoying behavior for us, given the co-development / monorepo setup - [tracked here](https://github.com/dart-lang/pub/issues/1867).

I'm about to do some unrelated work and will comment out before PRing, which should clear this up for now

**Comment by HofmannZ** [comment]
Should be fixed in #306
